crawler_mcp/crawlers/directory.py

```
@@ -249,49 +247,18 @@ def _is_valid_file(self, file_path: Path) -> bool:
             if file_path.stat().st_size == 0:
                 return False
 
-            # Skip very large files (>10MB) to prevent memory issues
-            if file_path.stat().st_size > 10 * 1024 * 1024:
+            # Skip large files based on configuration
+            max_size_bytes = settings.directory_max_file_size_mb * 1024 * 1024
```

Ruff (0.12.2)

Undefined name `settings`

(F821)